packages/create-instantsearch-app/src/templates/React InstantSearch/src/App.tsx.hbs

```diff
@@ -31,6 +31,8 @@ const searchClient = algoliasearch(
   '{{apiKey}}'
 );
 
+const future = { preserveSharedStateOnUnmount: true };
+
```

Comment:
Why outside of the app? It shouldn't create issues if inlined, no?

Reply:
inline objects can be annoying if we track the changes (which we don't yet do). Can put inline though

Reply:
It's fine as is. Good for me!